Madness/Parser.swift

```diff
@@ -105,19 +123,33 @@ func containsAt<C1: CollectionType, C2: CollectionType where C1.Generator.Elemen
 }
 
 // Returns a parser that satisfies the given predicate
+public func satisfy(pred: Character -> Bool) -> Parser<String.CharacterView, Character>.Function {
+	let updatePos = updatePosCharacter
+	let parser: Parser<String.CharacterView, Character>.Function = tokenPrim(updatePos, pred)
+	return parser
```

Comment:
Since `updatePosCharacter` can only be applied to collections of characters, I left the other one around for parsing arbitrary collections.